### DISCUSSION TITLE
System font

### DISCUSSION THREAD

**Discussion by Ab-Mr** [ImranR98/Obtainium#1032] [Ideas]
Option to use the system font

**Comment by roke-julian-lockhart** [comment]
https://github.com/ImranR98/Obtainium/issues/415#issue-1651085215

**Reply by roke-julian-lockhart** [reply]
@Ab-Mr, please mark https://github.com/ImranR98/Obtainium/discussions/1032#discussioncomment-7371376 as the answer.
